src/components/Checkout/DonateRepaymentsToggle.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-	<div v-if="showToggle" class="donate-repayments-toggle">
+	<div v-if="showToggle">
 		<kv-checkbox
-			class="donate-repayments-label"
+			class="tw-relative tw-cursor-pointer tw-leading-4 tw-pt-1 tw-pl-1 md:tw-text-right"
```

Comment:
Unless removing the leading class causes a problem, please remove it. Ideally line-heights are handled by our base typography rules.